pkg/cloud/services/eks/config.go

```diff
@@ -187,7 +187,11 @@ func (s *Service) createUserKubeconfigSecret(ctx context.Context, cluster *eks.C
 		return fmt.Errorf("creating base kubeconfig: %w", err)
 	}
 
-	execConfig := &api.ExecConfig{APIVersion: "client.authentication.k8s.io/v1alpha1"}
+	// Version v1alpha1 was removed in Kubernetes v1.23.
+	// Version v1 was released in Kubernetes v1.23.
+	// Version v1beta1 was selected as it has the widest range of support
+	// This should be changed to v1 once EKS no longer supports Kubernetes <v1.23
+	execConfig := &api.ExecConfig{APIVersion: "client.authentication.k8s.io/v1beta1"}
```

Comment:
Ah the last version was 1.19 with alpha, right? So this should be okay and not affect existing clusters.

Reply:
Looks like 1.10 was the last version with no beta and only alpha https://github.com/kubernetes/kubernetes/tree/release-1.10/staging/src/k8s.io/client-go/pkg/apis/clientauthentication
I doubt anyone is still using that version, and definitely would not be able to install a new cluster with it.